lib/auth/authentication_backend.rb

```diff
@@ -44,7 +44,7 @@ def create_group(name:, description:)
 
   # Returns a secret shared code to associate a TOTP app/device with
   def associate_device(access_token:)
-    associate = client.associate_software_token(access_token: access_token)
+    associate = client.associate_software_token(session: access_token)
```

Comment:
were we just using the key previously?

Reply:
This is an AWS method ... In this instance it takes an access_token which is different to a session_token.... So this one has to stay.